mne/viz/_3d.py

```diff
@@ -763,6 +964,13 @@ def plot_trans(info, trans='auto', subject=None, subjects_dir=None,
                                             opacity=alphas[key], figure=fig)
         if key != 'helmet':
             surface.actor.property.backface_culling = True
+    if brain and 'lh' not in surfs:  # one layer sphere
+        assert bem['coord_frame'] == FIFF.FIFFV_COORD_HEAD
+        center = bem['r0'].copy()
+        if coord_frame == 'mri':
+            center = apply_trans(head_mri_t, center)
```

Comment:
You aren't using the standard system here -- you should apply `head_trans` here (with no conditional). The current code won't work, for example, when the user plots in the MEG device coord frame.